consensus/polybft/fsm.go

```diff
@@ -108,6 +109,14 @@ func (f *fsm) BuildProposal() (*pbft.Proposal, error) {
 		return nil, err
 	}
 
+	// set the timestamp
```

Comment:
Do we need to move `headerTime` assignment above `extraData` assignment? It's strange because `extraParent, err := GetIbftExtra(parent.ExtraData)` is kept at its place

Reply:
Fixed [dad6eba](https://github.com/0xPolygon/polygon-edge/pull/796/commits/dad6ebaa45aa22820d59d8f3f3efde388220bbc9). Sorry for making such unnecessary change :pray:  :slightly_smiling_face: